### ISSUE TITLE
OTA blocks retransmission

### ISSUE THREAD

**Issue by ahmedwahdan** [aws/amazon-freertos#1458]
**Describe the bug**
The OTA agent has continuous receiving of duplicate OTA blocks.
In parallel to OTA agent task MQTT communications I send data to another topic every 5 seconds, and even if I disable the transmission this duplication occurs.
The communication is through greengrass 

**System information**
- Board : ESP32
- Operating System : Windows
- 08.2019
- Project [Greengrass+OTA]

**Screenshots or console output**
![OTA_Retransmission](https://user-images.githubusercontent.com/5406736/67568705-37025700-f72d-11e9-8499-4434203a1642.JPG)

![Duplication](https://user-images.githubusercontent.com/5406736/67569149-5a79d180-f72e-11e9-8989-3c336144b269.JPG)

![Duplication2](https://user-images.githubusercontent.com/5406736/67573670-2e178280-f739-11e9-9018-f56894277cd9.JPG)

Thank you!

**Comment by pvyawaha** [comment]
Hello,

Thank you for reporting this issue and sharing logs.

Please set the request timer to a higher value in the OTA agent config using the config parameter  otaconfigFILE_REQUEST_WAIT_MS . From the logs you shared it seems the request are being sent faster than the data blocks being received. You can also control the number of blocks requested by lowering the OTA config otaconfigMAX_NUM_BLOCKS_REQUEST . Please let me know how this works for you and if you have further questions.

**Comment by pvyawaha** [comment]
Hello, 

Please reopen this issue if you have more questions related to this or post a new issue for questions on other topics. Closing this issue now.
